pcapgo/read.go

```diff
@@ -149,6 +149,11 @@ func (r *Reader) Snaplen() uint32 {
 	return r.snaplen
 }
 
+// SetSnaplen sets the snapshot length of the capture file.
```

Comment:
Could you add a larger comment here discussing why you'd want to override the snaplen that's encoded in the file?  I get it from the pull request description, but I think it's worth specifically mentioning here to justify the existence of this function.

Reply:
You are right. I'll expand on this in the source code as well.